src/types/profile.js

```diff
@@ -208,6 +208,8 @@ export type ProfileMeta = {|
   preprocessedProfileVersion: number,
   appBuildID?: string,
   sourceURL?: string,
+  physicalCPUs?: number,
```

Comment:
Can we add comments as to why these might not exist? It would be nice to know if it's because older profiles don't have the information, or if it's platform/feature-specific information. Same for the gecko-profile types.

Reply:
It is the former, but yes, I will add comments.